### PR TITLE
Add fallback to EtherToken

### PR DESCRIPTION
**Pull request by abandeali1**
Adds a fallback to EtherToken which calls `deposit`.

### CODE REVIEW

File: contracts/tokens/EtherToken.sol [hunk 1]
```diff
@@ -17,9 +17,18 @@ contract EtherToken is StandardTokenWithOverflowProtection {
     /*
      *  Read and write functions
      */
+
+    /// @dev Send ether to contract to deposit.
```
Comment (fabioberger) [inline contracts/tokens/EtherToken.sol]: How about: `Fallback to calling deposit when ether is sent directly to contract`

File: contracts/tokens/EtherToken.sol [hunk 2]
```diff
@@ -29,13 +38,10 @@ contract EtherToken is StandardTokenWithOverflowProtection {
     /// @dev Sells tokens in exchange for Ether, exchanging them 1:1.
     /// @param amount Number of tokens to sell.
     function withdraw(uint amount)
-        external
+        public
```
Comment (fabioberger) [inline contracts/tokens/EtherToken.sol]: This could still be `external`. Any reason to switch it to `public`?

Comment (abandeali1) [inline contracts/tokens/EtherToken.sol]: Mostly for consistency with other contracts, we haven't really made use of any external methods.

File: test/ts/ether_token.ts [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+      } catch (err) {
+        testUtil.assertThrow(err);
+      }
+    });
+
+    it('should convert ether tokens to ether with sufficient balance', async () => {
+      const initEthTokenBalance = await ethToken.balanceOf(account);
+      const initEthBalance = await getEthBalance(account);
+      const ethTokensToWithdraw = initEthTokenBalance;
+      assert.equal(cmp(ethTokensToWithdraw, 0), 1);
```
Comment (fabioberger) [inline test/ts/ether_token.ts]: This is highly unreadable... Let's do:

```
assert.equal(ethTokensToWithdraw.toNumber(), 0);
```

File: test/ts/ether_token.ts [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+
+      const gasPrice = new BigNumber(web3Instance.toWei(20, 'gwei'));
```
Comment (fabioberger) [inline test/ts/ether_token.ts]: This constant is used in multiple tests. Let's declare it in the `contract()` scope above. This way we only declare it in one place.

**Pull request merged**